maat-court-data-api/src/main/java/gov/uk/courtdata/assessment/validator/CreatePassportAssessmentValidator.java

```diff
@@ -19,13 +19,9 @@
     @Override
     public Optional<Void> validate(CreatePassportAssessment createPassportAssessment) {
 
-        if (isBlank(createPassportAssessment.getNworCode())) {
-            throw new ValidationException("New work reason code is required");
-        } else if (isBlank(createPassportAssessment.getUserCreated())) {
+        if (isBlank(createPassportAssessment.getUserCreated())) {
             throw new ValidationException("Username is required");
-        } else if (isBlank(createPassportAssessment.getPastStatus())) {
-            throw new ValidationException("Past Status is required");
-        } if (createPassportAssessment.getFinancialAssessmentId() == null) {
+        } else if (createPassportAssessment.getFinancialAssessmentId() == null) {
```

Comment:
There should be a check to reject create requests with a passport assessment ID.

Reply:
There is no id field in the CreatePassportAssessment object, so cannot be checked